### PR TITLE
Fix untested javascript files in functional test

### PR DESCRIPTION
**Pull request by ongspxm**

### CODE REVIEW

**Comment by ongspxm** [comment]
https://github.com/reposense/RepoSense/blob/0029569a040f466229c8e6e0027f01ae6082d107/src/functional/java/reposense/Entry.java#L47

problem with the filetype checks, but fixing this exposes many other directory problems that was not caught before.

https://github.com/reposense/RepoSense/blob/0029569a040f466229c8e6e0027f01ae6082d107/src/functional/java/reposense/Entry.java#L57

will have to continue tomorrow, cos everything baozha-ing now :(

File: src/functional/java/reposense/Entry.java [hunk 1]
```diff
@@ -54,12 +54,14 @@ private void verifyAllJson(File expectedDirectory, String actualRelative) {
...
 
     private void assertJson(File expectedJson, String expectedPosition, String actualRelative) {
-        File actual = new File(actualRelative + expectedPosition);
+        File actual = new File(actualRelative + "/" + expectedPosition);
         Assert.assertTrue(actual.exists());
         verifyContent(expectedJson, actual);
     }
 
     private void verifyContent(File expected, File actual) {
+        System.out.println("checking "+actual+" with "+expected);
```
Comment (eugenepeh) [inline src/functional/java/reposense/Entry.java]: This message is redundant.

File: src/functional/resources/expected/summary.js [hunk 2]
```diff
@@ -1 +1 @@
...
...romDate":"2017-10-27","toDate":"2017-10-28"},{"insertions":0,"deletions":0,"fromDate":"2017-10-28","toDate":"2017-10-29"},{"insertions":0,"deletions":0,"fromDate":"2017-10-29","toDate":"2017-10-30"},{"insertions":69,"deletions":82,"fromDate":"2017-10-30","toDate":"2017-10-31"},{"insertions":317,"deletions":34,"fromDate":"2017-10-31","toDate":"2017-11-01"},{"insertions":35,"deletions":12,"fromDate":"2017-11-01","toDate":"2017-11-02"},{"insertions":0,"deletions":0,"fromDate":"2017-11-02","toDate":"2017-11-03"},{"insertions":0,"deletions":0,"fromDate":"2017-11-03","toDate":"2017-11-04"},{"insertions":0,"deletions":0,"fromDate":"2017-11-04","toDate":"2017-11-05"},{"insertions":0,"deletions":0,"fromDate":"2017-11-05","toDate":"2017-11-06"},{"insertions":0,"deletions":0,"fromDate":"2017-11-06","toDate":"2017-11-07"},{"insertions":180,"deletions":97,"fromDate":"2017-11-07","toDate":"2017-11-08"},{"insertions":0,"deletions":0,"fromDate":"2017-11-08","toDate":"2017-11-09"},{"insertions":0,"deletions":0,"fromDate":"2017-11-09","toDate":"2017-11-10"},{"insertions":109,"deletions":60,"fromDate":"2017-11-10","toDate":"2017-11-11"},{"insertions":246,"deletions":296,"fromDate":"2017-11-11","toDat
```
Comment (eugenepeh) [inline src/functional/resources/expected/summary.js]: explore why the insertion and deletion becomes 0.

**Comment by ongspxm** [comment]
done and ready to roll

File: src/functional/java/reposense/Entry.java (hunk 1 above)
Comment (ongspxm) [reply src/functional/java/reposense/Entry.java]: donez

File: src/functional/resources/expected/summary.js (hunk 2 above)
Comment (ongspxm) [reply src/functional/resources/expected/summary.js]: explored

**Comment by ongspxm** [comment]
consider this impossible.

- `ft_temp/<timestamp>/CS2103AUG2017-W09-B2_main/result.js` generates different results on different runs.

it is unlikely we will be able to write a functional test in this manner

**Comment by ongspxm** [comment]
put on hold for the moment

**Comment by eugenepeh** [comment]
Taken over to #106

**Pull request closed without merging**